index.js

```diff
@@ -13,16 +13,24 @@ app.get('/', function (req, res) {
 	res.send(`Group1`);
 });
 
+
 // read courses.json file
 const filePath = __dirname + '/data/courses.json';
-const content = fs.readFileSync(filePath, 'utf-8');
-const parsedContent = JSON.parse(content);
+
+// callback is added
+const content = fs.readFile(filePath, 'utf-8',(err,data)=>{
+	if(err){
+		console.error(err)
+	}
+	JSON.parse(data)
+});
+
```

Comment:
This function should be in (get and post  methods) otherwise you can access fs.readFile data  because they work async .